Source/Core/Core/Core.cpp

```diff
@@ -122,6 +130,13 @@ void FrameUpdateOnCPUThread()
     NetPlay::NetPlayClient::SendTimeBase();
 }
 
+void OnFrameEnd()
+{
+#ifdef USE_MEMORYWATCHER
+  s_memory_watcher->Step();
```

Comment:
This crashes when playing a fifolog because s_memory_watcher hasn't been initialized.

Reply:
Perhaps the memory watcher initialization should be moved from CpuThread to EmuThread?